nservicebus/messaging/forwarding.md

```diff
@@ -7,7 +7,18 @@ redirects:
 ---
 
 
-Use this feature to forward successfully processed messages from an endpoint to a specified destination endpoint. If there is a need to collect all successfully processed messages from all endpoints, then use the [Auditing Feature](/nservicebus/operations/auditing.md) instead.
+Use this feature to forward successfully processed messages from an endpoint to a specified destination endpoint. Forwarding messages is particularly useful in complex upgrade scenarios, when the old version and new version of a particular endpoint are running side-by-side.
+
+Note that forwarding is a solution to a specific problem and generally should be used for a limited transition period. In case of long-term need to duplicate messages the [Publish-Subscribe](/nservicebus/messaging/publish-subscribe/) pattern will be usually more appropriate.
```

Comment:
@indualagarsamy I removed that sentece, since I'm not 100% what are the possible usage scenarios I don't want the description to be too limiting. Do you think it'd make sense to suggest considering Pub-Sub or sagas as alternatives or that also assumes too much about how people might want to use that feature?